### ISSUE TITLE
Segmentation fault 

### ISSUE THREAD

**Issue by AlexTS1980** [martinkersner/train-CRF-RNN#45]
Both FCN and CRF-RNN report the segmentation fault error in the conv1_1 layer whenever I try finetuning with my own data. After some attempts I realized this is something to do with labels; when I comments out this layer, the network loads OK until the loss. I coundn't find any fix so far. Do you know anything about this? Data are images Bx3x250x250, labels are also images Bx1x250x250
